src/module/migration/migrations/722-crafting-system-data.ts

```diff
@@ -22,5 +22,9 @@ export class Migration722CraftingSystemData extends MigrationBase {
         if (!Array.isArray(crafting.formulas)) {
             crafting.formulas = [];
         }
+
+        if(!R.isNumber(crafting.repairValue)){
+            crafting.repairValue = 5;
+        }
```

Comment:
Instead of running this on an old migration we can just add the value during data prep. See `prepareBaseData()` in [module/actor/creature/document.ts](https://github.com/foundryvtt/pf2e/blob/8a404f752ea493e4313b0595860b2c3f6ffbce5e/src/module/actor/creature/document.ts#L284) though this one would go in character instead.

Reply:
I actually had that in there already, removing it from the migration one.